src/components/AddFormControl.js

```diff
@@ -0,0 +1,256 @@
+import React from 'react';
+import AddFormListing from './AddFormListing';
+import AddFormRequest from './AddFormRequest';
+import { connect } from 'react-redux';
+import { changeRoute } from '../actions/router.action';
+
+class AddFormControl extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      listingType: 'listing',
+      itemType: '',
+      title: '',
+      description: '',
+      categories: [],
+      price: 0,
+      images: [],
+      latitude: '',
+      longitude: '',
+      zipCode: '',
+      userId: '',
+      date_time_added: ''
+    };
+    this.listingChanged = this.listingChanged.bind(this);
+    this.requestChanged = this.requestChanged.bind(this);
+    this.captureData = this.captureData.bind(this);
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.submitHandler = this.submitHandler.bind(this);
+  }
+
+  captureData(inputType, data) {
+    alert(inputType);
+    switch (inputType) {
+      case 'title':
+        this.setState({ title: data });
+        break;
+    }
+  }
+
+  listingChanged() {
+    this.setState({
+      listingType: 'listing'
+    });
+  }
+
+  requestChanged() {
+    this.setState({
+      listingType: 'requesting'
+    });
+  }
+
+  submitHandler = event => {
+    event.preventDefault();
+    // alert('You are submitting');
+    console.log(event);
+
+    fetch(
+      'https://cors-anywhere.herokuapp.com/https://api-dot-pacific-plating-262123.appspot.com/api/v1.0/items/categories',
+      {
+        method: 'GET'
+      }
+    )
+      .then(response => {
+        return response.json();
+      })
+      .then(jsonData => {
+        console.log(jsonData);
+      });
+
+    fetch(
+      'https://cors-anywhere.herokuapp.com/https://api-dot-pacific-plating-262123.appspot.com/api/v1.0/items/postnew',
+      {
+        method: 'POST',
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          listingType: this.state.listingType,
+          itemType: this.state.itemType,
+          title: this.state.title,
+          description: this.state.description,
+          price: this.state.price,
+          date_time_added: Date(),
+          test: 'yes'
+        })
+      }
+    );
+  };
+
+  handleInputChange(event) {
```

Comment:
This function is just setting an arbitrary value based on which component called it, the actual value is lost.